### DISCUSSION TITLE
Using livewire on structured laravel

### DISCUSSION THREAD

**Discussion by habe90** [livewire/livewire#4909] [1. Help]
Hi, 
is a posbile uses livewire on module/package structured laravel?
![Screenshot_1](https://user-images.githubusercontent.com/22656146/164750334-dbe4e64f-567f-4cec-a7af-7691516394e7.png)
![Screenshot_2](https://user-images.githubusercontent.com/22656146/164750337-9283af27-6d8d-427f-b95f-86a8c431ddb5.png)

**Comment by denizozturk** [comment]
It's possible to do this, but getting out of the "resources" directory can be a bit of a hassle.

Livewire config (config/livewire.php) file should be like the below;
```
'view_path' => base_path()
```

View config (config/view.php)  file should be like the below;
```
'paths' => [
    resource_path('views'),
    base_path(),
]
```

Run this command;
```
dnz@ozturk ..ments/GitHub/livewire-playground % php artisan make:livewire platform/themes/components/button
COMPONENT CREATED  🤙

CLASS: app/Http/Livewire/Platform/Themes/Components/Button.php
VIEW:  platform/themes/components/button.blade.php
```


After creating this component, you need to edit the view path in the render method. You have to do this after creating each new component, the default path is incorrect.
```
public function render()
    {
        return view('***.***.***.***.***.platform.themes.components.button');
    }
```

![screen](https://user-images.githubusercontent.com/34531750/164896081-abbe2479-412f-4f4b-ac78-3bbb6dddf7f2.png)

That's it, hope it helps
